prisma/seedAll.ts

```
@@ -0,0 +1,100 @@
+
+  if (truncate) {
+    if (process.env.NODE_ENV === 'production' && process.env.ALLOW_SEED_TRUNCATE !== 'true') {
+      console.error('Refusing to truncate database in production. Set ALLOW_SEED_TRUNCATE=true to override.');
+      process.exit(1);
+    }
+
+    console.log('Truncating seed-target tables (conservative list)...');
+    // Use a Postgres TRUNCATE with CASCADE for speed and to avoid FK ordering issues.
+    // Keep this list conservative: only tables that may be created by seeding/demo scripts.
+    const tables = [
+      'webhooks',
```

ESLint

[error] Replace `'webhooks'` with `"webhooks"`

(prettier/prettier)